tslearn/docs/gettingstarted.rst

```diff
@@ -85,8 +85,8 @@ To read from / write to this format, have a look at the :ref:`tslearn.utils <mod
 Playing with your data
 ----------------------
 
-Once your data is loaded and formatted according to `tslearn` standards, the next step is to feed machine learning
-models with it. Most `tslearn` models inherit from `scikit-learn` base classes, hence interacting with them is very
+Once your data is loaded and formatted according to `tslearn` standards, the next step is to feeding to machine learning
```

Comment:
This is grammatically incorrect. `The next step is to feed it to machine learning models` or the original would be better.